articles/migrate/tutorial-migrate-vmware-agent.md

```diff
@@ -64,6 +64,13 @@ Assign the Virtual Machine Contributor role to the account, so that you have per
 - Write to an Azure managed disk.
 
 
+### Assign permissions to register the Replication Appliance in Azure AD
+
+If you are following the least privilege principle, assign the **Application Developer** Azure AD role to the user registering the Replication Appliance. Follow the [Assign administrator and non-administrator roles to users with Azure Active Directory](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal) guide to do so.
```

Comment:
```suggestion
### Assign permissions to register the Replication Appliance in Azure AD

If you are following the least privilege principle, assign the **Application Developer** Azure AD role to the user registering the Replication Appliance. Follow the [Assign administrator and non-administrator roles to users with Azure Active Directory](/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal) guide to do so.
```
absolute links to docs.microsoft.com articles aren't allowed 
removing loc "en-us"